iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/utils/IcebergCatalogUtil.java

```diff
@@ -99,8 +99,11 @@ private static KerberosClient initKerberosAndReturnClient(
       Map<String, String> properties, Configuration conf) {
     try {
       KerberosClient kerberosClient = new KerberosClient(properties, conf);
-      File keytabFile =
-          kerberosClient.saveKeyTabFileFromUri(Long.valueOf(properties.get("catalog_uuid")));
+
+      // For Iceberg rest server, we haven't set the catalog_uuid, so we set it to 0 as there is
+      // only one catalog in the rest server, so it's okay to set it to 0.
+      String catalogUUID = properties.getOrDefault("catalog_uuid", "0");
```

Comment:
If supporting multi catalogs , how to handle this?

Reply:
I need more information on how we will support multiple catalogs so I can make the necessary changes.